poly/constants.go

```diff
@@ -0,0 +1,15 @@
+package poly
+
+import (
+	"github.com/dedis/crypto/edwards"
+)
+
+const (
```

Comment:
What are these consts for exactly, and how do you intend them to be used?  It looks like they're only used in testing?  Anything that is only used in testing should be in a _test.go file.  And in any case, I don't really see the benefit of using vague terms like MINIMUM (minimum of what set of possibilities) or MODERATE for picking test configurations; why not just list a few actual configurations to test (e.g., specific t and n thresholds, specific ciphersuites, etc.)?